vault/core.go

```diff
@@ -2361,26 +2429,10 @@ func (s standardUnsealStrategy) unseal(ctx context.Context, logger log.Logger, c
 		return err
 	}
 
-	if c.getClusterListener() != nil {
-		if err := c.setupRaftActiveNode(ctx); err != nil {
-			return err
-		}
-
-		if err := c.startForwarding(ctx); err != nil {
-			return err
-		}
-
-	}
-
-	c.clusterParamsLock.Lock()
-	defer c.clusterParamsLock.Unlock()
-
-	c.metricsCh = make(chan struct{})
-	go c.emitMetricsActiveNode(c.metricsCh)
-
-	// Establish version timestamps at the end of unseal on active nodes only.
-	if err := c.handleVersionTimeStamps(ctx); err != nil {
-		return err
+	// Finally, start processing invalidations. We'll have cleared the queue
+	// when we started this, but any
+	if c.standby.Load() {
```

Comment:
I'm fairly sure this doesn't work, because we're never marked active until later and so we'll always be standby here even on active nodes, but I have a commit to clean this up in my PostgreSQL invalidation branch. I think its fine to deal with later as we have other safeguards against populating this map.